resources/views/components/image-button-list.blade.php

```diff
@@ -13,6 +13,9 @@
                 @else
                     @image($image['relative_url'], $image['title'])
                 @endif
+            @elseif(!empty($image['relative_url']) && !empty($image['secondary_image']))
+                <img src="/promos/{{ $image['promo_group_id'] }}/{{ $image['secondary_image']}}">
```

Comment:
Should this be updated with the new `secondary_relative_url` value?

Reply:
I think we're deleting this file, I didn't mean to commit this change